src/components/AnimesList.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+/* eslint-disable react/jsx-props-no-spreading */
+import 'bootstrap/dist/css/bootstrap.min.css';
+import {
+  Col, Container, Row, Tooltip, OverlayTrigger,
+} from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchAnimes } from '../redux/animes';
+import Anime from './Anime';
+import Header from './Header';
+
+const AnimesList = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchAnimes());
+  }, []);
+  const AnmiesState = useSelector((state) => state.anmiesReducer.animes);
+
+  const renderTooltip = (props) => (
+    <Tooltip id="button-tooltip" {...props}>
+      Click To See Facts
+    </Tooltip>
+  );
```

Comment:
- [ ] Please, kindly add a component that allows the user to filtered using given parameters, e.g. category name.

> Uses a Filter stateless component to filter data retrieved from the API.

Reply:
done